web/connect/result/error.html

```diff
@@ -20,9 +20,12 @@
       href="/static/favicon-16x16.png"
     />
     <link rel="manifest" href="/static/site.webmanifest" />
-    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <link
```

Comment:
do we really need a different css for each integration ?
aren't we plan to move all this UI code to the standard UI ?

Reply:
I already merged all `error.html` and `error.js` files. It's possible to do the same for the CSS files, but it requires more manual testing because they're not identical.